ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+TabManagerDelegate.swift

```diff
@@ -189,6 +195,9 @@ extension BrowserViewController: TabManagerDelegate {
     tab.downloadDelegate = self
     tab.certificateStore = profile.certStore
     attachTabHelpers(to: tab)
+    if let braveShieldsTabHelper = tab.braveShieldsHelper {
+      tab.addPolicyDecider(braveShieldsTabHelper)
```

Comment:
This already happens in `attachTabHelpers`

Reply:
Ah good catch, missed removing it here when I moved it there.